src/routes/(pages)/app/(app)/components/default-explore-page.svelte

```diff
@@ -84,22 +84,45 @@
       ],
     }[PUBLIC_NETWORK] ?? [];
 
+  const FEATURED_PROJECTS_ACCOUNT_IDS =
+    {
+      1: [
+        '80927338512810702724070905882237017022089417038277884279346528518144', // svelte
+        '80924437970685862336445237697146051810361983738382071874368862945280', // libgit2
+        '80926915932044874567662239576164593128211047641556385561345492254720', // rollup
+        '80926893484859255783823398448697429050946431358357802763937929756672', // ratatui
+        '80907185984472938178947231143934051564334108789056821384881454972928', // gitoxide
+        '80914552162673449833728329632154702021826817315592132431697405804544', // tanstack-query
+        '80928551845018914533911124940323675679718022636490642073491274203136', // vitest
+        '80928956680761126169933069488824219882230509123653596947248123478016', // websockets
+      ],
+    }[PUBLIC_NETWORK] ?? [];
+
   export let projects: DefaultExplorePageFeaturedProjectFragment[];
   export let blogPosts: z.infer<typeof postsListingSchema>;
   export let featuredDripLists: DefaultExplorePageFeaturedDripListsFragment[];
   export let totalDrippedPrices: Awaited<ReturnType<typeof cachedTotalDrippedPrices>>;
   export let tlv: number;
 
+  $: featuredWeb3Projects = projects.filter((p) =>
+    FEATURED_WEB_3_PROJECTS_ACCOUNT_IDS.includes(p.account.accountId),
+  );
+
   $: featuredProjects = projects.filter((p) =>
-    FEATURED_PROJECT_ACCOUNT_IDS.includes(p.account.accountId),
+    FEATURED_PROJECTS_ACCOUNT_IDS.includes(p.account.accountId),
   );
 
   // 2 latest posts. Sort by date
   $: blogPosts = blogPosts
     .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
     .slice(0, 2);
 
-  $: recentlyClaimedProjects = projects.slice(-4);
+  // I (jason) sometimes claim projects for testing purposes. crude way to get rid of those
```

Comment:
😅 Maybe we should all follow the same pattern cause I use random keystrokes. 🤠

Reply:
haha yeah. maybe

or we build #1242 :D

Reply:
working on it. going well 🤓